tldw_Server_API/app/core/Slides/slides_db.py

```
@@ -319,6 +393,119 @@ def _insert_version_snapshot(self, conn: sqlite3.Connection, row: PresentationRo
+        total = int(count_row["cnt"]) if count_row else 0
+        return [VisualStyleRow(**dict(row)) for row in rows], total
+
+    def update_visual_style(
+        self,
+        *,
+        style_id: str,
+        name: str,
+        style_payload: str,
+    ) -> VisualStyleRow:
+        if not name:
+            raise InputError("name is required")
```

Ruff (0.15.6)

[warning] Avoid specifying long messages outside the exception class

(TRY003)